### PR TITLE
Fix-3225: used BudgetUpdatedEvent instead of BudgetSetEvent

### PR DESCRIPTION
**Pull request by mkbeefcake**
Fixed the issue #3225 

image

### CODE REVIEW

**Review by chrlschwb — APPROVED** [review]

File: packages/ui/src/council/queries/council.graphql [hunk 1]
```diff
@@ -252,6 +258,9 @@ query GetPastCouncilWorkingGroups($fromBlock: Int!, $toBlock: Int!) {
   newMissedRewardLevelReachedEvents(where: { inBlock_lte: $toBlock }, orderBy: [inBlock_DESC]) {
```
Comment (thesan) [inline packages/ui/src/council/queries/council.graphql:258]: Shouldn't `newMissedRewardLevelReachedEvents` also be restricted to the council term:
```suggestion
  newMissedRewardLevelReachedEvents(where: { inBlock_gte: $fromBlock, inBlock_lte: $toBlock }) {
```
Or am I missing something ?

Also I think `budgetSetEvents` should be removed, or is this query is used somewhere else ?

Comment (mkbeefcake) [reply packages/ui/src/council/queries/council.graphql:258]: I checked budgetSetEvents is not used in the source code. saying about newMissedRewardLevelReachedEvents, that was the original code, so i will follow you.

Comment (mkbeefcake) [reply packages/ui/src/council/queries/council.graphql]: I updated the #3225

**Review by thesan — APPROVED** [review]
Thanks

**Comment by ivanturlakov** [comment]
Tested on https://pioneer-2.vercel.app/#/council/past-councils/0000000j
Mainnet


✅ The budget columns look good
✅ The final budget matches the figure from the consuls' report

Screenshot 2023-10-03 at 15 23 24


Screenshot 2023-10-03 at 14 24 53

**Comment by ivanturlakov** [comment]
I haven't figured out how to check if the data in the first two columns is correct yet. Checked the reports of the consuls and group leaders for the corresponding term, but unfortunately, there are no such figures there

**Comment by ivanturlakov** [comment]
The page still needs correction, the data marked does not correspond to the data in the reports of consuls and working group leaders

Screenshot 2023-10-11 at 14 16 11

**Comment by chrlschwb** [comment]
@ivanturlakov the `total spent` and `total spent on proposals` fields are going to be corrected in another PR
https://github.com/Joystream/pioneer/pull/4554
So, this PR should only fix the below fields

**Pull request merged**